Tests/TestingApps/PaywallsTester/PaywallsTester/UI/Views/OfferingsList.swift

```diff
@@ -13,67 +13,104 @@ import RevenueCatUI
 #endif
 import SwiftUI
 
+// TODO: Ask Barbara about how to present
 struct OfferingsList: View {
-
-    fileprivate struct Template: Hashable {
-        var name: String?
-    }
-
-    fileprivate struct Data: Hashable {
-        var sections: [Template]
-        var offeringsBySection: [Template: [Offering]]
+    
+    private struct OfferingPaywall: Hashable {
+        let offering: OfferingsResponse.Offering
+        let paywall: PaywallsResponse.Paywall
     }
-
+    
```

Comment:
I'd recommend setting up Xcode's `Automatically trim trailing whitespace` to avoid these

<img width="942" alt="Screenshot 2024-04-17 at 8 52 40 PM" src="https://github.com/RevenueCat/purchases-ios/assets/3922667/0dc16345-adb1-492c-87ed-3f97f614b2ae">